realm/realm-library/src/main/cpp/util.cpp

```diff
@@ -365,11 +365,29 @@ static string string_to_hex(const string& message, StringData& str, const char*
     return ret.str();
 }
 
+static string error_code_to_message(size_t error_code){
+    switch (error_code){
```

Comment:
Where did you find the description of these errors? Perhaps add a comment linking to the source of that?

Reply:
https://github.com/realm/realm-core/blob/ebb051525d4ddcd0ea6e0400ec25f0c0901f6c61/src/realm/util/utf8.hpp#L260